runtime/src/tests.rs

```diff
@@ -192,6 +192,26 @@ fn pallet_zksync_availability() {
     });
 }
 
+#[test]
+fn pallet_risc0_availability() {
+    new_test_ext().execute_with(|| {
+        let dummy_origin = AccountId32::new([0; 32]);
+
+        let dummy_vk = [0u8; 32];
+        let dummy_proof = vec![];
+        let dummy_pubs = vec![];
+
+        assert!(SettlementRisc0Pallet::submit_proof(
```

Comment:
Add your test here

Reply:
And also the availability one

Reply:
Added test for pallet availability and weight linking.